docs/uart.rst

```diff
@@ -10,17 +10,19 @@ a serial interface.
 Functions
 =========
 
-.. method:: init(baudrate=9600, bits=8, parity=None, stop=1, pins=None)
+.. method:: init(baudrate=9600, bits=8, parity=None, stop=1, \*, tx=None, rx=None)
```

Comment:
@deshipu thanks for the heads up... I'd never seen that before. You learn something new every day... ;-)